src/main/java/org/mozilla/jss/asn1/ASN1Util.c

```diff
@@ -53,45 +53,44 @@
  *        (together 2 bytes)
  */
 JNIEXPORT jstring JNICALL
-Java_org_mozilla_jss_asn1_ASN1Util_getTagDescriptionByOid(JNIEnv *env, jobject this, jbyteArray oidBA)
-{
-    SECItem *oid = NULL;
-    SECOidTag oidTag = SEC_OID_UNKNOWN;
-    const char *oidDesc = NULL;
-    jstring description = (jstring)"";
+Java_org_mozilla_jss_asn1_ASN1Util_getTagDescriptionByOid(JNIEnv *env,
+		jobject this, jbyteArray oidBA) {
+	SECItem *oid = NULL;
+	SECOidTag oidTag = SEC_OID_UNKNOWN;
+	const char *oidDesc = NULL;
+	jstring description = (jstring) "";
 
-    if (oidBA == NULL) {
-        JSS_throwMsg(env, INVALID_PARAMETER_EXCEPTION,
-            "JSS getTagDescriptionByOid: oidBA null");
-        goto finish;
-    } else {
-        /**************************************************
-         * Setup the parameters
-         *************************************************/
-        oid = JSS_ByteArrayToSECItem(env, oidBA);
-        if (oid == NULL) {
-            JSS_throwMsg(env, INVALID_PARAMETER_EXCEPTION,
-                "JSS getTagDescriptionByOid: JSS_ByteArrayToSECItem failed");
-            goto finish;
-        }
+	if (oidBA == NULL) {
+		JSS_throwMsg(env, INVALID_PARAMETER_EXCEPTION,
+				"JSS getTagDescriptionByOid: oidBA null");
+		goto finish;
```

Comment:
This code seems to be over indented.

Reply:
Though I agee that the code looks over indented there is something odd.
I'm confused because the pull request is titled "Autoformat .java, .c, .h file in org.mozilla.crypto" and the branch shown at the top is ckelleyRH:autoformat_jss_crypto yet the file changes shown here are for src/main/java/org/mozilla/jss/asn1/ASN1Util.c which is inside the asn1 subdirectory, how come?
I actually did a checkout of the fork and switched to to branch and don't see this changes!